### PR TITLE
0.8.1 release

### PR DESCRIPTION
**Pull request by zhujun98**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -31,9 +31,10 @@
 def find_version():
     with open(osp.join('extra_foam', '__init__.py')) as fp:
         for line in fp:
```
Comment (ebadkamil) [inline setup.py]: Why not read entire file in a string ` f.read()`

`m = re.search(r'^__version__ = "(\d+\.\d+\.\d[a-z]*\d*)"', f.read(), re.M)`

Comment (zhujun98) [reply setup.py]: The `re` search is line by line. `^__version__` is for the start of a line.

Comment (zhujun98) [reply setup.py]: Resolved in offline discussion. Both solutions work.

**Comment by zhujun98** [comment]
Thanks for the review!

**Pull request merged**